qa/suites/fs/upgrade/upgraded_client/tasks/2-upgrade.yaml

```diff
@@ -0,0 +1,63 @@
+teuthology:
+  premerge: |
+            local branch = yaml.teuthology.branch
+            local upgrade_branch = ""
+            local osd_release = ""
+            if tostring(branch) == "nautilus" then
+              upgrade_branch = "pacific"
+              osd_release = "ceph osd require-osd-release pacific"
+            elseif tostring(branch) == "pacific" then
+              upgrade_branch = "reef"
+              osd_release = "ceph osd require-osd-release reef"
```

Comment:
This is applied _after_ the upgrade, yes? I wonder if the failures you see with p->r upgrade have something to do with this.

Reply:
it then had to fail with nautilus->pacific, also this is the same upgrade yaml found in any other suite

Reply:
OK. I've not yet gone through the failed job yet. Will have a look.